src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/SummaryEvidence.cshtml

```diff
@@ -36,14 +35,22 @@
                     <span class="govuk-!-font-size-36 govuk-!-font-weight-bold govuk-!-padding-left-0"> <span class="govuk-body">Submitted evidence notes for review &nbsp;</span> </span>
```

Comment:
@GuidoWorkDev  Possible to remove some of the commented out HTML in these two files?

Reply:
comments cleaned up